src/main/java/org/corfudb/infrastructure/SequencerServer.java

```diff
@@ -258,6 +256,7 @@ public void reset() {
     @Override
     public void reboot() {
```

Comment:
Again, what is the reboot function for? I can't see this being used for anything other than test, and even then I can only see it as bad practice for tests. If anything, create a new SequencerServer and shutdown this one.